vhs-frontend/.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+cd vhs-frontend
```

Comment:
This line shouldn't be necessary.

Reply:
If I remove that line, I get this error:
`error Couldn't find a package.json file in "/home/ipetk/Projects/vhs-frontend-lab"`

So I followed https://typicode.github.io/husky/#/?id=custom-directory

Reply:
I checked, I was wrong actually. We use this on the project as well so forget my "correction". :)